HandsOnTutorial/1-hello-world.fsx

```diff
@@ -1,15 +1,19 @@
 ﻿(*** hide ***)
-#load "credentials.fsx"
-
+#load "Thespian.fsx"
+#load "Azure.fsx"
 
 open System
 open System.IO
 open MBrace.Core
-open MBrace.Azure
 open MBrace.Flow
 
+// Initialize client object to an MBrace cluster:
+let cluster = 
+//    getAzureClient() // comment out to use an MBrace.Azure cluster; don't forget to set the proper connection strings in Azure.fsx
+    initThespianCluster(4) // use a local cluster based on MBrace.Thespian; configuration can be adjusted using Thespian.fsx
```

Comment:
Agreed, it is awkward indeed. I chose this as a lesser of many evils. My intention here was to avoid hiding any cluster initialization side-effects inside the scripts.